docs/src/main/asciidoc/getting-started-reactive.adoc

```diff
@@ -252,7 +272,7 @@ Then, edit the `ReactiveGreetingResource` class to match the following content:
 
 [source, java]
 ----
-package org.acme.getting.started;
+package org.acme.quickstart;
```

Comment:
Is it really the package we want?

Reply:
it's the package generated with the command line given above. 
It seems that codestart is changing the package names.

Reply:
We have `-DclassName="org.acme.quickstart.ReactiveGreetingResource"` so that's expected :)

But maybe we should rather fix the `className`?

Reply:
(asking that because the quickstart itself is indeed using the `org.acme.getting.started` package name)

Reply:
yes, let's fix the classname

Reply:
Fixed - reverted to the previous package name and updated the command lines